src/app/signUp/page.tsx

```diff
@@ -34,6 +37,43 @@ export default function SignUpPage() {
     console.log(data);
   };
 
+  const handleSignUp = (): any => {
+    const email = getValues("email");
+    const password = getValues("password");
+    const name = getValues("name");
+    const employment = getValues("employment");
+    const degree = getValues("studyingDegree");
+    if (
+      degree === "" ||
+      degree === undefined ||
+      selectedEducation === "" ||
+      selectedEducation === undefined ||
+      lookingForWork === undefined ||
+      name === "" ||
+      name === undefined ||
+      employment === "" ||
+      employment === undefined
+    ) {
+      return toast.error("Form needs to be filled");
+    }
+    createWithEmailAndPassword(email, password).then(
```

Comment:
Can you double check if the error path syntax correct?

It's usually common to write this as (using promise chaining)

```
createWithEmailAndPassword(...)
    .then((resp) => { ... })
    .catch((err) => { ... })
```

Rather than having two parameters in the `then` method